mw-webapp/src/utils/auth/writeNewUserCredentials.ts

```diff
@@ -9,7 +9,7 @@ export const writeNewUserCredentials = async () => {
       return;
     }
     if (userCredentials.user.email && userCredentials.user.displayName) {
-      UserService.writeNewUserData(
+      UserService.addUser(
```

Comment:
what about using `create, update, delete, get` prefixes for CRUD operations

Reply:
Should I use read instead of get? ot getUsers is okey?

Reply:
Done